pages/index.tsx

```diff
@@ -120,25 +120,28 @@ export default class extends React.Component<IProps, IState> {
       seconds: '',
       mouseInteraction:
         this.props.showLink == null ? false : this.props.showLink,
-      lastTickHadColon: true
+      lastTickHadColon: false
```

Comment:
what's the purpose of this? it will start without a colon but both colons will still blink at the same time

Reply:
This makes the colons appear when the seconds change. Before they disappeared when the seconds changed.

Reply:
ahhh that makes sense, ok I'll merge this one and think about making them alternate, thanks for the PR!